docs/workforce-management-framework/topics/pro-running-the-demo-app.adoc

```diff
@@ -86,6 +86,4 @@ Open a new terminal and navigate to the folder containing _raincatcher-angularjs
 npm run start
 ----
 
-Both the Mobile and Portal Demo open automatically in a browser.
-
-For login details, refer to xref:credentials-for-the-demo-application-{chapter}[Credentials for the Demo Application] section of xref:ref-demo-app-{chapter}[{WFM-RC-NameShort} Demo Application Reference Material].
```

Comment:
This text was relocated into the user story AND effectively appears in the exact same location as ^.